### PR TITLE
Adds documentation for the lutron Caseta devices

### PR DESCRIPTION
**Pull request by gurumitts**
**Description:**
Adds documentation for the Lutron Caseta devices. 

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

### CODE REVIEW

File: source/_components/lutron_caseta.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: "Lutron_Caseta"
```
Comment (Landrash) [inline source/_components/lutron_caseta.markdown]: Remove the _ from the title.

File: source/_components/lutron_caseta.markdown [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+Configuration variables:
+
+- **host** (*Required*): The IP address of the Lutron SmartBridge.
+- **user** (*Required*): The login name of the user. The user `lutron` always exists.
+- **password** (*Required*): The password for the user specified above. `integration` is the password for the always-present `lutron` user.
+
+<p class='note'>
+  It is recommended to assign a static IP address to your Lutron SmartBridge. This ensures that it won't change IP addresses, so you won't have to change the `lutron_ip` if it reboots and comes up with a different IP address.
```
Comment (Landrash) [inline source/_components/lutron_caseta.markdown]: Singular on "addresses".

File: source/_components/lutron_caseta.markdown [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+footer: true
+logo: lutron.png
+ha_category: Hub
+featured: False
+ha_release: 0.40
+---
+
+[Lutron](http://www.lutron.com/) is an American lighting control company. They have several lines of home automation devices that manage light switches/dimmers, occupancy sensors, HVAC controls, etc. The `lutron_caseta` component in Home Assistant is responsible for communicating with the Lutron SmartBridge for these systems.
+
+This component only supports the Caseta line of products.  Current only supports dimmers as Home Assistant lights.
```
Comment (Landrash) [inline source/_components/lutron_caseta.markdown]: Double space between sentences.

File: source/_components/lutron_caseta.markdown [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+description: "Instructions how to use Lutron_Caseta devices with Home Assistant."
+date: 2017-01-28 13:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lutron.png
+ha_category: Hub
+featured: False
+ha_release: 0.40
```
Comment (Landrash) [inline source/_components/lutron_caseta.markdown]: Update version to 0.41 or later if that's more likely to be the version it's merged into.

**Comment by gurumitts** [comment]
Thanks... Will make the changes soon.  I need to change the config section of the docs as well.

**Comment by gurumitts** [comment]
Hello, Are there any more requested changes?  Can we merge this in?  The component was included in the 0.41 release.

**Pull request merged**